src/Schema/AbstractSchema.php

```diff
@@ -574,4 +574,42 @@ public function getViewNames(string $schema = '', bool $refresh = false): array
 
         return (array) $this->viewNames[$schema];
     }
+
+    /**
```

Comment:
No need for PHPDoc since it implements interface and there phpdoc already exists.